apps/oc_chef_wm/itest/oc_chef_wm_cookbook_artifact_version_SUITE.erl

```diff
@@ -35,14 +35,38 @@ init_per_suite(Config) ->
 end_per_suite(Config) ->
     setup_helper:base_end_per_suite(Config).
 
+init_per_testcase(http_delete, Config) ->
+    OrgId = ?config(org_id, Config),
+    NewChecksum = <<"b804944c17edc107073d6a1f27aa3842">>,
+    ok = chef_sql:mark_checksums_as_uploaded(OrgId,
+                                             [NewChecksum]),
+
+    Config2 = [{new_checksum, NewChecksum} | Config],
+    init_per_testcase(generic, Config2);
 init_per_testcase(_, Config) ->
+    OrgId = ?config(org_id, Config),
+
     ok = meck:new(chef_s3),
     ok = meck:expect(chef_s3, generate_presigned_url, 5,
                      fun(_, _, _, Checksum, _) ->
                          <<"http://fake.url/for/", Checksum/binary>>
                      end),
+    ok = meck:expect(chef_s3, delete_checksums, 2,
```

Comment:
Looks like this is only for http_delete, so should probably be in its own init_per_testcase for that case.